<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>postscreen</class>
 <widget class="QDialog" name="postscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>332</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Share your location</string>
  </property>
  <widget class="QPushButton" name="btn_post">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;
background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Post</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="text">
       <string>Name of location</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="input_name">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_type">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>School</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coffe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restaurant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Park</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mall</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Market</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hospital</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Others</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="input_address">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>22</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22pt &quot;Source Code Pro&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Share your location</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
